src/model/document.ts

```diff
@@ -88,7 +88,7 @@ export class Document implements Feature {
       this.identifiers.add(this.url);
     }
     this.kinds.add(`${this.parsedDocument.type}-document`);
-    // this._resolve();
+    this.warnings = base.warnings.slice();
```

Comment:
see the removal of `get warnings()` below. @rictic is it ever possible for warnings to be added to a ScannedDocument after the Document has been created from it?